src/Pages/AddRecipe/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { FormEvent, useContext } from "react";
+import NumberInput from "../../Utilities/NumberInput";
+import TextInput from "../../Utilities/TextInput";
+import SubmitInput from "../../Utilities/SubmitInput";
+import Header from "../../Components/Header";
+import BASE_URL from "../../settings";
+import UserContext from "../../Contexts/UserContext";
+import Button from "../../Utilities/Button";
+import { Link } from "react-router";
+import { useNavigate } from "react-router-dom";
+import TextAreaInput from "../../Utilities/TextArea";
+
+function AddRecipe() {
+  const { userId } = useContext(UserContext);
+  const navigate = useNavigate();
+
+  async function addRecipeData(formData: FormData) {
+    let data = {
+      prep_time: 0,
+      cook_time: 0,
+      name: formData.get("name"),
+      instructions: formData.get("instructions"),
+      ingredients: [],
+    };
+    if (formData.get("prep_time") != "") {
+      data.prep_time = Number(formData.get("prep_time"));
+    }
+    if (formData.get("cook_time") != "") {
+      data.cook_time = Number(formData.get("cook_time"));
+    }
```

Comment:
```suggestion
    if (formData.get("prep_time") != "") {
      data.prep_time = Math.floor(Number(formData.get("prep_time")));
    }
    if (formData.get("cook_time") != "") {
      data.cook_time = Math.floor(Number(formData.get("cook_time")));
    }
```
Let's only keep whole minutes.

Reply:
done